Library/Formula/antlr.rb

```diff
@@ -13,4 +11,23 @@ def install
       java -classpath #{prefix}/antlr-#{version}-complete.jar:. org.antlr.v4.runtime.misc.TestRig "$@"
     EOS
   end
+
+  test do
+    path = testpath/"Expr.g4"
+    path.write <<-EOS.undent
+    grammar Expr;
+    prog:\t(expr NEWLINE)* ;
+    expr:\texpr ('*'|'/') expr
+        |\texpr ('+'|'-') expr
+        |\tINT
+        |\t'(' expr ')'
+        ;
+    NEWLINE :\t[\\r\\n]+ ;
+    INT     :\t[0-9]+ ;
+    EOS
+    ENV["CLASSPATH"] = ".:#{prefix}/antlr-#{version}-complete.jar:#{ENV["CLASSPATH"]}"
```

Comment:
prepend_path uses `File::PATH_SEPARATOR`: https://github.com/jacknagel/homebrew/blob/1d9db84b604e44c153926e27c4c6dfdc74d480e4/Library/Homebrew/extend/ENV/shared.rb#L81-83

I see now that it checks that the argument is a directory, though, which is probably why it didn't work.